karma.conf.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function(config) {
+    config.set({
+
+        basePath: '',
+        frameworks: ['browserify', 'jasmine'],
+        files: [
+            'bower_components/angular/angular.js',
+            'bower_components/angular-animate/angular-animate.js',
```

Comment:
no problem, I just thought it would be more appropriate to use the source files. I will update this.